internal/version/version.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The cert-manager Authors
+Copyright 2023 The cert-manager Authors.
```

Comment:
Why 2023?

Reply:
This was an unrelated change, which has now been reversed. Thanks!